trackman/templates/library/index.html

```diff
@@ -18,6 +18,14 @@ <h1>Library</h1>
     <li class="nav-item"><a class="nav-link" href="{{ url_for('trackman_library.fixup') }}">Fixup</a></li>
 </ul>
 
+<!-- TODO beautify -->
+<h2>search<h2>
+    <p>Only searches exact matches</p>
+    <form action=artist>
+	<label for="artist">Artist: </label><input type="text" id="artist" name="artist"/>
```

Comment:
Please move the space in between the label and input. Also, the input is missing Bootstrap classes; please refer to other forms for an example.